CreeDictionary/CreeDictionary/relabelling.py

```diff
@@ -0,0 +1,200 @@
+from __future__ import annotations
+
+import csv
+from enum import IntEnum
+from typing import Iterable, Optional, TextIO, Tuple, TypeVar, Union
+
+from utils import shared_res_dir
+from utils.types import FSTTag, Label
+
+CRK_ALTERNATE_LABELS_FILE = shared_res_dir / "crk.altlabel.tsv"
+
+Default = TypeVar("Default")
```

Comment:
By itself this is very mysterious. What kind of default is this, and for what?

Reply:
This is a generic type variable for the return type of the default argument of `.get(key, default)`. Honestly, Python's generics type variables are really gross! And this function doesn't need to be generic, so I'll just delete it!